docs/Using Randomness/chainlink-vrf.md

```diff
@@ -19,7 +19,7 @@ metadata:
 
 # Generate Random Numbers in your Smart Contracts
 
-Chainlink VRF (Verifiable Random Function) is a provably-fair and verifiable source of randomness designed for smart contracts. Smart contract developers can use Chainlink VRF as a tamper-proof RNG to build reliable smart contracts for any applications which rely on unpredictable outcomes:
+Chainlink VRF (Verifiable Random Function) is a provably-fair and verifiable source of randomness designed for smart contracts. Smart contract developers can use Chainlink VRF as a tamper-proof random number generator (RNG) to build reliable smart contracts for any applications which rely on unpredictable outcomes:
```

Comment:
I think it should be capitalized: `Random Number Generator`

Reply:
Do you all have a "words and phrases" list of some sort? 

I disagree, but not enough to push back if you'd like to submit a PR to capitalize it.

My reasoning is that RNG isn't really a Chainlink "proper noun," or something you would want users to pay attention to and remember.

Reply:
Ok sorry I guess I do care lol. 

It looks like it's not capitalized here: https://blog.chain.link/chainlink-vrf-on-chain-verifiable-randomness/

Reply:
Verifiable Random Function is a proper noun, but random number generator is a more generic term. This can stay lower-case.